.github/workflows/test-release.yml

```diff
@@ -69,9 +69,60 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
-        os: [ubuntu-latest, macos-latest, windows-latest]
-        dist: [wheel, sdist]
+        python-version: ['3.7', '3.11', '3.12']
+        python-architecture: ['x64']
```

Comment:
'python-architecture' is not used in the setup-python action, so it can be dropped from the matrix. It might be a good idea to test macos both aarch64 and x64, but if there's only one arch using the default is IMO sufficient.

Reply:
For the ci in general or for this workflow/wheel-based ci?